source/pool/test/integration.js

```diff
@@ -293,8 +294,58 @@ describe('Pool Integration Tests', () => {
         )
       ).to.be.revertedWith('INVALID_TOKEN')
     })
-  })
 
+    it('withdrawWithSignature does not revert with signer who is admin', async () => {
```

Comment:
I think these should actually go in the unit test file, as these functions are not interacting with any other contract.